macro/run_simScript.py

```diff
@@ -44,8 +44,9 @@
 
 globalDesigns = {'2016':{'dy':10.,'dv':5,'ds':7,'nud':1,'caloDesign':0,'strawDesign':4},\
                  '2018':{'dy':10.,'dv':6,'ds':9,'nud':3,'caloDesign':3,'strawDesign':10},\
-                 '2022':{'dy':8.,'dv':6,'ds':9,'nud':3,'caloDesign':3,'strawDesign':10}}
-default = '2022'
+                 '2022':{'dy':8.,'dv':6,'ds':9,'nud':3,'caloDesign':3,'strawDesign':10}, \
+                 '2023':{'dy':8.,'dv':6,'ds':9,'nud':4,'caloDesign':3,'strawDesign':10}}
+default = '2023'
```

Comment:
This will certainly cause a merge conflict with #420 , but I can take care of it there.